Cargo.toml

```diff
@@ -14,7 +17,7 @@ name = "glow"
 path = "src/lib.rs"
 
 [features]
-default = ["web-sys"]
+default = []
```

Comment:
Could we keep `web-sys` as default? In the long term I think all crates will probably switch from `stdweb` to `web-sys`, but `stdweb` support is still useful at the moment

Reply:
I don't understand why either `web-sys` or `stdweb` should be default. Neither is needed by people who use `glow` on native, right?

Reply:
Right, native doesn't need `web-sys`, but `web-sys` and the other `wasm32-unknown-unknown` dependencies aren't built on native in the current setup as far as I know.

`web-sys` is gated by `[target.'cfg(target_arch = "wasm32")'.dependencies.web_sys]` in Cargo.toml, so it shouldn't actually get built when doing `cargo build` (instead only through `cargo build --target wasm32-unknown-unknown`).

e.g. I don't see `web-sys` in the dependency list when doing `cargo clean && cargo build` which seems to confirm this

Reply:
Ok, thank you for explaining!
If web-sys is indeed the way to go in the future, it seems reasonable to have it by default now, and eventually make it the only path.